[2022-06-19 00:18:03,290: DEBUG] Starting new HTTPS connection (1): webdevblog.ru:443
[2022-06-19 00:18:13,302: ERROR] Connection failed, try again!
[2022-06-19 00:19:27,968: DEBUG] Starting new HTTPS connection (1): webdevblog.ru:443
[2022-06-19 00:19:29,735: DEBUG] https://webdevblog.ru:443 "GET /logging-v-python/ HTTP/1.1" 200 None
[2022-06-19 00:23:58,014: DEBUG] Starting new HTTPS connection (1): webdevblog.ru:443
[2022-06-19 00:23:59,362: DEBUG] https://webdevblog.ru:443 "GET /logging-v-python/ HTTP/1.1" 200 None
[2022-06-19 00:23:59,417: ERROR] The directory with page's files exists yet!
[2022-06-19 00:25:43,337: DEBUG] Starting new HTTPS connection (1): webdevblog.ru:443
[2022-06-19 00:25:44,744: DEBUG] https://webdevblog.ru:443 "GET /logging-v-python/ HTTP/1.1" 200 None
[2022-06-19 00:25:44,802: ERROR] The directory with page's files exists yet!
[2022-06-19 00:26:01,165: DEBUG] Starting new HTTPS connection (1): webdevblog.ru:443
[2022-06-19 00:26:02,732: DEBUG] https://webdevblog.ru:443 "GET /logging-v-python/ HTTP/1.1" 200 None
2022-06-19 00:28:09,472 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): webdevblog.ru:443
2022-06-19 00:28:11,192 - urllib3.connectionpool - DEBUG - https://webdevblog.ru:443 "GET /logging-vython/ HTTP/1.1" 404 None
2022-06-19 00:29:43,785 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): webdevblog.ru:443
2022-06-19 00:29:45,073 - urllib3.connectionpool - DEBUG - https://webdevblog.ru:443 "GET /logging-v-python/ HTTP/1.1" 200 None
2022-06-19 00:35:03,480 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): webdevblog.ru:443
2022-06-19 00:35:04,748 - urllib3.connectionpool - DEBUG - https://webdevblog.ru:443 "GET /logging-v-python/ HTTP/1.1" 200 None
2022-06-19 00:35:04,857 - page_loader.logs.logs_config - DEBUG - that's a new page - /home/ulitka/python-project-lvl3/webdevblog-ru-logging-v-python.html
2022-06-19 00:35:04,858 - page_loader.logs.logs_config - DEBUG - that's a new dir for page's resources - /home/ulitka/python-project-lvl3/webdevblog-ru-logging-v-python_files
2022-06-19 00:35:04,858 - page_loader.logs.logs_config - DEBUG - successfull downloading /home/ulitka/python-project-lvl3/webdevblog-ru-logging-v-python.html
2022-06-19 01:09:08,064 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): webdevblog.ru:443
2022-06-19 01:09:09,628 - urllib3.connectionpool - DEBUG - https://webdevblog.ru:443 "GET /logging-v-python/ HTTP/1.1" 200 None
2022-06-19 01:09:09,736 - page_loader.logs.logs_config - DEBUG - that's a new page - /home/ulitka/python-project-lvl3/webdevblog-ru-logging-v-python.html
2022-06-19 01:09:09,737 - page_loader.logs.logs_config - DEBUG - that's a new dir for page's resources - /home/ulitka/python-project-lvl3/webdevblog-ru-logging-v-python_files
2022-06-19 01:09:09,737 - page_loader.logs.logs_config - DEBUG - successfull downloading /home/ulitka/python-project-lvl3/webdevblog-ru-logging-v-python.html
2022-06-19 01:09:46,751 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): webdevblog.ru:443
2022-06-19 01:09:47,457 - urllib3.connectionpool - DEBUG - https://webdevblog.ru:443 "GET /logging-v-pytn/ HTTP/1.1" 404 None
2022-06-19 01:09:47,463 - page_loader.logs.logs_config - DEBUG - that's a new page - /home/ulitka/python-project-lvl3/webdevblog-ru-logging-v-pytn.html
2022-06-19 01:09:47,463 - page_loader.logs.logs_config - DEBUG - that's a new dir for page's resources - /home/ulitka/python-project-lvl3/webdevblog-ru-logging-v-pytn_files
2022-06-19 01:09:47,464 - page_loader.logs.logs_config - DEBUG - successfull downloading /home/ulitka/python-project-lvl3/webdevblog-ru-logging-v-pytn.html
2022-06-19 01:21:09,123 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): webdevblog.ru:443
2022-06-19 01:21:10,953 - urllib3.connectionpool - DEBUG - https://webdevblog.ru:443 "GET /logging-v-pytn/ HTTP/1.1" 404 None
2022-06-19 01:21:10,958 - page_loader.logs.logs_config - WARNING - Something went wrong, a response code is 404
2022-06-19 01:21:10,960 - page_loader.logs.logs_config - DEBUG - that's a new page - /home/ulitka/python-project-lvl3/webdevblog-ru-logging-v-pytn.html
2022-06-19 01:21:10,961 - page_loader.logs.logs_config - DEBUG - that's a new dir for page's resources - /home/ulitka/python-project-lvl3/webdevblog-ru-logging-v-pytn_files
2022-06-19 01:21:10,961 - page_loader.logs.logs_config - DEBUG - successfull downloading /home/ulitka/python-project-lvl3/webdevblog-ru-logging-v-pytn.html
2022-06-19 01:22:01,223 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): webdevblog.ru:443
2022-06-19 01:22:01,969 - urllib3.connectionpool - DEBUG - https://webdevblog.ru:443 "GET /logging-v-pytn/ HTTP/1.1" 404 None
2022-06-19 01:22:01,973 - page_loader.logs.logs_config - WARNING - Something went wrong, a response code is 404
2022-06-19 01:22:01,974 - page_loader.logs.logs_config - DEBUG - that's a new page - /home/ulitka/python-project-lvl3/webdevblog-ru-logging-v-pytn.html
2022-06-19 01:22:01,975 - page_loader.logs.logs_config - DEBUG - that's a new dir for page's resources - /home/ulitka/python-project-lvl3/webdevblog-ru-logging-v-pytn_files
2022-06-19 01:22:01,975 - page_loader.logs.logs_config - DEBUG - successfull downloading /home/ulitka/python-project-lvl3/webdevblog-ru-logging-v-pytn.html
2022-06-19 01:23:18,417 - urllib3.connectionpool - DEBUG - Starting new HTTPS connection (1): webdevblog.ru:443
2022-06-19 01:23:19,168 - urllib3.connectionpool - DEBUG - https://webdevblog.ru:443 "GET /logging-v-pytn/ HTTP/1.1" 404 None
2022-06-19 01:23:19,174 - page_loader.logs.logs_config - WARNING - Something went wrong, a response code is 404
2022-06-19 01:25:42,147 - DEBUG - Starting new HTTPS connection (1): webdevblog.ru:443
2022-06-19 01:25:42,873 - DEBUG - https://webdevblog.ru:443 "GET /logging-v-pytn/ HTTP/1.1" 404 None
2022-06-19 01:25:42,876 - WARNING - Something went wrong, a response code is 404
2022-06-19 01:25:42,878 - ERROR - The directory with page's files exists yet!
2022-06-19 01:26:12,257 - DEBUG - Starting new HTTPS connection (1): webdevblog.ru:443
2022-06-19 01:26:13,795 - DEBUG - https://webdevblog.ru:443 "GET /logging-v-python/ HTTP/1.1" 200 None
